charts/keycloak/templates/servicemonitor.yaml

```diff
@@ -0,0 +1,25 @@
+{{ if and .Values.prometheus.operator.enabled }}
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: {{ template "keycloak.fullname" . }}
+  {{- if .Values.prometheus.operator.serviceMonitor.namespace }}
+  namespace: {{ .Values.prometheus.operator.serviceMonitor.namespace }}
```

Comment:
Do we really need/want this I'd tend to say that the `ServiceMonitor` should be in the same namespace as anythin else.

Reply:
I don't personally use it. It's there because some other charts like to deploy the service monitors to the "monitoring" namespace or similar. I'll remove it for now unless someone requests it.